src/libraries/Microsoft.Extensions.Logging.Console/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <IncludePlatformAttributes>true</IncludePlatformAttributes>
```

Comment:
Because this project has targets lower than 5.0 i needed to add platform attributes, it hasn't `Directory.Build.props` so i added one, but not sure if it is OK